tutorials/appengine-python-falcon/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Falcon API on App Engine Standard Environment
+description: Learn how to build a Falcon API in the Google App Engine standard environment.
+author: archelogos
+tags: App Engine, Python, Falcon, API
+date_published: 2017-04-27
+---
+This tutorial shows how to build a Python api with [Falcon][falcon].
+
+Falcon is a high-performance Python framework for building cloud APIs. It encourages the REST architectural style, and tries to do as little as possible while remaining highly effective.
+
+In order to follow this guide, you will need to install Python in your local machine.
+
+[python]: https://www.python.org/
+[falcon]: https://falconframework.org/
+
+## Objectives
+
+1. Create a Python app that uses falcon as a framework.
+2. Run the app locally.
+3. Deploy the Python app to Google App Engine standard environment.
+
+## Costs
+
+This tutorial does not use billable components of Google Cloud Platform so
+you do not need to enable the billing for your project to complete this tutorial.
+
+## Before you begin
+
+1.  Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
```

Comment:
Suggestion: "Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/), **and make note of the project ID**."